backend/api/v1/role_service.go

```diff
@@ -203,6 +211,20 @@ func (s *RoleService) getProjectUsingRole(ctx context.Context, role string) (boo
 	return false, 0, nil
 }
 
+func (s *RoleService) checkWorkspaceUsingRole(ctx context.Context, role string) (bool, error) {
```

Comment:
There could be project IAM policy using the role as well. I will write the SQL query later, 1 sec.

Reply:
```
SELECT EXISTS (
    SELECT 1
    FROM policy
    CROSS JOIN LATERAL jsonb_array_elements(payload->'bindings') AS binding
    WHERE type = 'bb.policy.iam'
      AND binding->>'role' = 'roles/projectDeveloper'
);
```

\O/